git/git.go

```diff
@@ -95,6 +91,16 @@ type Ref struct {
 	Sha  string
 }
 
+func (r *Ref) Refspec() string {
```

Comment:
👍 I agree with your comment that `Refspec` is a better name here. I think we can merge this as-is and I'll update my branch to pull in `Refspec()` over `String()`.